common/lib/xmodule/xmodule/course_module.py

```diff
@@ -34,6 +34,15 @@
 CATALOG_VISIBILITY_NONE = "none"
 
 
+def get_proctoring_list():
+    from django.conf import settings
```

Comment:
Can we somehow use `@XBlock.needs('settings')` instead? Or add these settings to `Advanced Settings`?
https://github.com/edx/xblock-utils/wiki/Settings-and-theme-support